src/content/docs/plugin/upload.mdx

```diff
@@ -80,7 +80,7 @@ import { upload } from '@tauri-apps/plugin-upload';
 upload(
   'https://example.com/file-upload',
   './path/to/my/file.txt',
-  (progress, total) => console.log(`Uploaded ${progress} of ${total} bytes`), // a callback that will be called with the upload progress
+  (uploading) => console.log(`Uploaded ${uploading.progress} of ${uploading.total} bytes`), // a callback that will be called with the upload progress
```

Comment:
Can we make this the same as below? So like this:
```suggestion
  ({ progress, total }) => console.log(`Uploaded ${progress} of ${total} bytes`), // a callback that will be called with the upload progress
```

Reply:
like that can also be done